packages/fuchsia_ctl/lib/src/ssh_client.dart

```diff
@@ -10,6 +11,9 @@ import 'package:process/process.dart';
 
 import 'operation_result.dart';
 
+/// The default timeout in milliseconds for the ssh command.
+const int defaultSshTimeoutMs = 5 * 60 * 1000;
```

Comment:
Maybe make this a `static const` member of `SshClient` and then it can just  be dropped from `image_paver.dart`.

Reply:
Done.